tactis/model/decoder.py

```diff
@@ -356,6 +444,8 @@ def loss(
         hist_true_u: torch.Tensor,
         pred_encoded: torch.Tensor,
         pred_true_u: torch.Tensor,
+        num_series: int,
+        num_timesteps: int,
```

Comment:
These are not used anywhere but in an assert, so they could safely be removed, together with the strange code which create them in the calling functions.

Reply:
Resolved.